src/System.Linq.Expressions/src/System/Linq/Expressions/Compiler/LambdaCompiler.Lambda.cs

```diff
@@ -136,9 +136,33 @@ private void EmitDelegateConstruction(LambdaExpression lambda)
             EmitDelegateConstruction(impl);
         }
 
-        private static Type[] GetParameterTypes(LambdaExpression lambda)
+        private static Type[] GetParameterTypes(LambdaExpression lambda, Type firstType)
         {
-            return lambda.Parameters.Map(p => p.IsByRef ? p.Type.MakeByRefType() : p.Type);
```

Comment:
Change https://github.com/dotnet/corefx/pull/11308 also removes the need for `Map` in `FindMethod` and `FindProperty`. Once both PRs are merged, the `Map` method can likely be removed altogether.